stable/prisma/ci/example-values.yaml

```diff
@@ -0,0 +1,57 @@
+# ------------------------------------------------------------------------------
+# Prisma:
+# ------------------------------------------------------------------------------
+
+replicas: 1
+
+image:
+  repository: prismagraphql/prisma
+  tag: 1.15-heroku
```

Comment:
Are you aware of this issue [here](https://github.com/prisma/prisma/issues/2850) re: scaling Prisma to multiple instances? I have not looked at your chart implementation and architecture design decisions as I do not have the best understanding of Kubernetes, but just wanted to make you aware of the specifics required to run Prisma in a horizontally scalable architecture.

Reply:
Jumping into the conversation 😄 I would like to help on this one, once this first version will be merged. Be careful tho that you for now you need to use the prisma-prod image for this, which doesn't have the `envsubst` hook the manage your config.

So, you will probably have to either build your own image with a custom similar hook, or wait for [https://github.com/prisma/prisma/issues/2869](this issue) to be solved and hope that the merged version will have an heroku version.

Reply:
Yeah, you are right. Once this is in, and there is something "tangible", we will be able to tackle issues faster and make the chart better by having more contributors (including Prisma people perhaps). Everything will be sorted out with time.